### PR TITLE
docs(di): Edits to initial draft. 

### PR DESCRIPTION
**Pull request by naomiblack**
See https://github.com/angular/angular...

### CODE REVIEW

File: modules/angular2/src/di/binding.js [hunk 1]
```diff
@@ -97,14 +101,16 @@ export class Binding {
    *
    * expect(injector.get(String)).toEqual('Hello');
    * ```
+   *
+   * @exportedAs angular2/di
```
Comment (mhevery) [inline modules/angular2/src/di/binding.js]: you dont need to put `exportedAs` on members. Only top level classes.

File: modules/angular2/src/di/binding.js [hunk 2]
```diff
@@ -97,14 +101,16 @@ export class Binding {
...
    */
   toValue;
 
   /**
-   * Bind a key to an alias of an existing key.
+   * Binds a key to the alias for an existing key.
    *
-   * An alias means that we will return the same instance as if the alias token was used. (This is in contrast to
-   * `toClass` where a separet instance of `toClass` will be returned.)
+   * An alias means that Angular returns the same instance as if the alias token was used. This is in contrast to
```
Comment (mhevery) [inline modules/angular2/src/di/binding.js]: DI, is Angular independent, can not use angular in any of the descriptions.

File: modules/angular2/src/di/binding.js [hunk 3]
```diff
@@ -170,14 +178,16 @@ export class Binding {
...
+   * function takes the resolved value. This shows that the [Injector] delays executing the `String` factory
+   * until after the `Number` is resolved. This can only be done if the `token` is retrieved.
+   *
+   * @exportedAs angular2/di
    */
   toAsyncFactory:Function;
 
   /**
-   * Used in conjunction with `toFactory` or `toAsyncFactory` and specifies the `token`s which should be injected
+   * Used in conjunction with `toFactory` or `toAsyncFactory` and specifies the `token` objects which should be injected
```
Comment (mhevery) [inline modules/angular2/src/di/binding.js]: Its just tokens no need for token objects. Everything is an object.

File: modules/angular2/src/di/annotations.js [hunk 4]
```diff
@@ -1,7 +1,7 @@
 import {CONST} from "angular2/src/facade/lang";
 
 /**
- * A parameter annotation that creates a synchronous eager dependency.
+ * A parameter annotation that creates a synchronous arc-eager dependency.
```
Comment (vicb) [inline modules/angular2/src/di/annotations.js]: @naomiblack what does arc-eager means ?

**Pull request closed without merging**